NEWS.md

```diff
@@ -52,6 +52,9 @@ This section lists changes that do not have deprecation warnings.
 
   * `broadcast` now treats `Ref` (except for `Ptr`) arguments as 0-dimensional
     arrays ([#18965]).
+  * `broadcast` now handles missing data (`Nullable`s) allowing operations to
+    be lifted over `Nullable`s, as if the `Nullable` were like an array with
+    zero or one element. ([#16961]).
```

Comment:
It's a breaking change, because `Nullable`s that were treated like scalars in some cases before are now treated like collections. To keep `Nullable`s like scalars you need to use the `Ref(x)` or `(x,)` trick now.